app/presenters/order_detail_notice_presenter.rb

```diff
@@ -0,0 +1,85 @@
+class OrderDetailNoticePresenter < DelegateClass(OrderDetail)
+
```

Comment:
The main advantage from my perspective is brevity. We can use method calls with an implicit `self` within the class. So we can just call `in_review?` rather than `order_detail.in_review?`.